WeakAurasOptions/InformationOptions.lua

```diff
@@ -143,27 +143,24 @@ function OptionsPrivate.GetInformationOptions(data)
     end
   end
 
-  -- Show warnings only for single selection for now
-  if not isGroup then
-    local icon, title, message = OptionsPrivate.Private.AuraWarnings.FormatWarnings(data.uid)
-    if title and message then
-      args.warningTitle = {
-        type = "header",
-        name = title,
-        width = WeakAuras.doubleWidth,
-        order = order,
-      }
-      order = order + 1
+  local icon, title, messages = OptionsPrivate.Private.AuraWarnings.FormatWarnings(data.uid)
```

Comment:
You want a **not isTmpGroup** check here and change the old comment to something like:

  -- Warnings
  -- One Aura/Group: Show warning of aura/group
  -- Multi-selection: Nothing

The story behind that is, that for the information tab almost every setting behaves in a non-standard way. E.g. url is set on everything selected, whereas description is only edited on one aura. That doesn't fit the old common options code, and I didn't want to add even more baggage to it.

So the information code is very bare-bones and does the merging itself with some duplicated code. The idea being that some day in the future I'll rewrite it to something better and then port the other pages to a new common infrastructure...